pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -111,8 +112,20 @@ func getBinding(iface *v1.Interface, domain *api.Domain) (BindMechanism, error)
 		return nil, err
 	}
 
+	populateMacAddress := func(vif *VIF, iface *v1.Interface) error {
+		if iface.MacAddress != "" {
+			macAddress, err := net.ParseMAC(iface.MacAddress)
+			if err != nil {
+				return err
+			}
+			vif.MAC = macAddress
+		}
+		return nil
+	}
+
 	if iface.Bridge != nil {
 		vif := &VIF{Name: podInterface}
+		populateMacAddress(vif, iface)
```

Comment:
can't we set the mac address in case of slirp?

Reply:
We can but we don't need to proxy it through our struct and just extract it from v1.interface when constructing qemu args.

Reply:
@yuvalif see https://github.com/kubevirt/kubevirt/pull/1213/commits/d61c1671037fef382a71181d407d68aa220cae25

Reply:
ok. got it